### PR TITLE
Update index.html & Chinese Traditional Translation

### PR DESCRIPTION
**Pull request by open3**
## Description
1. Update the icons (include favicon) to the newer one.
2. Add Chinese Traditional Translation
3. Add a drop-down menu for language selection (index.html)
4. Update sidebar.md for language selection (document)

## How Has This Been Tested?
Running my GitHub Pages (https://open3.github.io/files-community.github.io)

## Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by chingucoding** [comment]
We recently merged a PR that updated the icon so you need to merge with main so we can merge this PR.

**Comment by open3** [comment]
@chingucoding Fixed!

**Comment by chingucoding** [comment]
@SiderealArt Would you like to take a look regarding the translations, my Chinese skills are nonexistent 😅

File: zh-Hant/docs/articles/replace-file-explorer.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# 開啟項目時預設啟動 Files
+
+*由於此方法涉及到系統文件，請確保事先新建還原點並自行承擔風險，且此方法並不適用於所有人。*
+
+**使用登錄項目修改登錄檔**
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/replace-file-explorer.md:5]:
```suggestion
**使用腳本自動修改**
```

**Comment by chingucoding** [comment]
Thank you for your help @SiderealArt ! Let me know if there is anything else that should be adjusted or if this is good to be merged! 😄

**Comment by SiderealArt** [comment]
I've been quite busy recently, could you allow me a few more days? I'll tag you if finished. Sorry for that!

**Comment by chingucoding** [comment]
Of course @SiderealArt no stress, take all the time you need! Thank you so much for your help!

File: zh-Hant/docs/articles/code-style.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+[5.](#5) 請不要在任何名稱中使用大寫字母。
+
+[6.](#6) 請使用預設的資料類型名稱，例如 `int`、`string` 而非 `Int32`、`String`。
+
+[7.](#7) 請為每個 Interface 的前輟打上 `I`。
+
+[8.](#8) 請垂直對齊每個大括號。
+
+[9.](#9) 請不要將 Enum 得後輟命名為 `Enum` 或 `Flag(s)`。
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:25]:
```suggestion
[9.](#9) 請不要將 Enum 的後輟命名為 `Enum` 或 `Flag(s)`。
```

File: zh-Hant/docs/articles/code-style.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+[10.](#10) 請使用 `Is`、`Has`、`Have`、`Any`、`Can`... 以此類推 作為 Boolean 前輟。
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:27]:
```suggestion
[10.](#10) 請使用 `Is`、`Has`、`Have`、`Any`、`Can`... 等類似名稱作為布林值名稱的前輟。
```

File: zh-Hant/docs/articles/code-style.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# 程式風格準則
+
+[1.](#1) 請使用 `PascalCase`:
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:3]:
```suggestion
[1.](#1) 請使用小駝峰式命名法 (PascalCase):
```

File: zh-Hant/docs/articles/code-style.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- 於類別（Class）名稱中
+- 於 method 名稱中
+- 於 const 名稱中
+
+[2.](#2) 請使用 `camelCase`:
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:8]:
```suggestion
[2.](#2) 請使用大駝峰式命名法 (camelCase):
```

File: zh-Hant/docs/articles/code-style.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- 於 method 參數中
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:9]:
```suggestion
- 於方法參數中
```

File: zh-Hant/docs/articles/code-style.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# 程式風格準則
+
+[1.](#1) 請使用 `PascalCase`:
+- 於類別（Class）名稱中
+- 於 method 名稱中
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/code-style.md:5]:
```suggestion
- 於方法名稱中
```

File: zh-Hant/docs/articles/configure-win-e.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
+# Win + E 快捷鍵配置
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/configure-win-e.md:1]:
```suggestion
# 如何使用 Win + E 快捷鍵來開啟 Files
```

File: zh-Hant/docs/articles/configure-win-e.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+**方法一：使用 AutoHotkey 腳本**
+
+您可以透過使用 [AutoHotKey](https://www.autohotkey.com/) 腳本配置 Win + E：
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/configure-win-e.md:5]:
```suggestion
您可以使用 [AutoHotKey](https://www.autohotkey.com/) 腳本配置 Win + E：
```

File: zh-Hant/docs/articles/custom-themes.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
+# 客製化主題
+
+Files 允許您修改應用程式中的 xaml 文件，這代表使用者們可以完全修改掉 Files 的外觀。
+
+客製化主題將從 Files 主程式資料夾中的 `LocalState\Themes` 資料夾載入，並透過「外觀」設定選單的一處下拉式選單中選擇您的主題。
+
+### 快速上手
+
+1. 開啟記事本，打上下方的主題樣板。完成後將文件另存新檔並儲存至 `%userprofile%\AppData\Local\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState\Themes\test1.xaml`。
```
Comment (SiderealArt) [inline zh-Hant/docs/articles/custom-themes.md:9]:
```suggestion
1. 開啟記事本，貼上下方的主題樣板。完成後將文件另存新檔並儲存至 `%userprofile%\AppData\Local\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState\Themes\test1.xaml`。
```

**Comment by SiderealArt** [comment]
Hi @chingucoding , sorry to bother you, but seems like @open3 isn't responding for a while now, maybe you can merge this PR and I'll open another to fix the issue? Or should we keep waiting? Please let me know your thought. Thanks again!

**Comment by chingucoding** [comment]
Hi @SiderealArt, thank you for your ping and your help! I think we could wait a couple more days (maybe until it's been two weeks) and then merge the PR. My plan would be to merge your changes into this PR then and then merge the PR, that way you don't have to create a new PR. If you wish to do this differently, let me know @SiderealArt !

**Review by chingucoding — APPROVED** [review]
Thank you @open3 and @SiderealArt for your help and efforts with this PR! LGTM!

**Pull request merged**